### DISCUSSION TITLE
Streaming only a single window?

### DISCUSSION THREAD

**Discussion by omigeot** [LizardByte/community#51] [Sunshine] [answered]
I'm looking for ways to only stream a single windows. Like for instance, define an application as a given game, and ensure it starts only streams THAT game (wether it's fullscreen or not), by it's X11 id, I guess.

So far, the only way to do that I've been able to find is using a nested X server (like Xephyr), but unless I'm mistaken it means I won't have HW OpenGL/Vulkan in it.

**Comment by ReenigneArcher** [comment, marked as answer]
It's not possible with sunshine currently.

**Reply by omigeot** [reply]
Noted, thanks. Is it somewhat on the roadmap, or is it just too far from the project's scope?

**Reply by ReenigneArcher** [reply]
It's not on any roadmap. The project is open source and driven by contributions from the community.

With that said, I've recently been playing with screen capture in python and it seems pretty simple to capture a single window. It is probably just as simple in c++, but I am not an expert on this.
